common/app/model/EmailSubscriptions.scala

```diff
@@ -363,8 +363,8 @@ object EmailSubscriptions {
       teaser = "Get up to speed on the most interesting and provoking issues and join the debate every afternoon",
       description = "Guardian Opinion's daily email newsletter with the most shared opinion, analysis and editorial articles from the last 24 hours — sign up to read, share and join the debate every afternoon.",
       frequency = "Weekday afternoons",
-      listId = "2313",
-      subscribedTo = subscribedListIds.exists{ x => x == "2313" || x == "3811" },
```

Comment:
If 3814 is the other variant, you will probably want tones applied: https://www.theguardian.com/email/form/plaintone/3814

#15488 should cover all the places that need updating